accord-core/src/main/java/accord/api/KeyRange.java

```diff
@@ -127,6 +136,45 @@ public boolean containsKey(K key)
         return compareKey(key) == 0;
     }
 
+
+    /**
+     * Returns a negative integer, zero, or a positive integer if both points of the provided range are less than, the
+     * reange intersects this range, or both points are greater than this range
+     */
+    public int compareIntersecting(KeyRange<K> that)
+    {
+        if (this.start.compareTo(that.end) >= 0)
+            return -1;
```

Comment:
Shouldn't this by +1 if we're keeping consistency with normal `compareTo` semantics?

Reply:
oops, fixed